pyproject.toml

```
@@ -0,0 +1,46 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "FreeSimpleGUI"
+version = "5.2.0"
+description = "The free-forever Python GUI framework."
+readme = {file = "README.md", content-type = "text/markdown"}
```

In the original setup.cfg, I believe this was a typo. Probably why the description doesn't show on PyPI. Let's correct it here.

```suggestion
readme = {file = "readme.md", content-type = "text/markdown"}
```

Alternatively, we can rename `readme.md` to `README.md`.